docs/config.md

```diff
@@ -57,15 +57,16 @@ for more details.
-| `OTEL_DOTNET_AUTO_LOGS_ENABLED_INSTRUMENTATIONS`     | Comma-separated list of logs source instrumentations you want to enable. Set to `none` to disable all metric instrumentations.                                                                                                                                  | all available instrumentations |
-| `OTEL_DOTNET_AUTO_LOGS_DISABLED_INSTRUMENTATIONS`    | Comma-separated list of logs source instrumentations you want to disable.                                                                                                                                                                                       |                                |
+| Environment variable                                   | Description                                                                                                                                                                                                                                                     | Default value                                              |
+|--------------------------------------------------------|-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|------------------------------------------------------------|
+| `OTEL_DOTNET_AUTO_INTEGRATIONS_FILE`                   | List of bytecode instrumentations JSON configuration filepaths, delimited by the platform-specific path separator (`;` on Windows, `:` on Linux and macOS). For example: `%ProfilerDirectory%/integrations.json`. It is required for bytecode instrumentations. |                                                            |
+| `OTEL_DOTNET_AUTO_INSTRUMENTATION_ENABLED`             | Enables all instrumentations.                                                                                                                                                                                                                                   | `true`                                                     |
+| `OTEL_DOTNET_AUTO_TRACES_INSTRUMENTATION_ENABLED`      | Enables all trace instrumentations. Overrides `OTEL_DOTNET_AUTO_INSTRUMENTATION_ENABLED`.                                                                                                                                                                       | Same as `OTEL_DOTNET_AUTO_INSTRUMENTATION_ENABLED`         |
+| `OTEL_DOTNET_AUTO_TRACES_{0}_INSTRUMENTATION_ENABLED`  | Configuration pattern for enabling or disabling a specific trace instrumentation, where `{0}` is the case-sensitive name of the instrumentation you want to enable. Overrides `OTEL_DOTNET_AUTO_TRACES_INSTRUMENTATION_ENABLED`.                                | Same as `OTEL_DOTNET_AUTO_TRACES_INSTRUMENTATION_ENABLED`  |
```

Comment:
I would consider calling out explicitly that it also overrides `OTEL_DOTNET_AUTO_INSTRUMENTATION_ENABLED`. Same comment applies to other places.

Reply:
@theletterf?

Reply:
Ok with that!